src/genie/libs/parser/ironware/tests/ShowMPLSLSP/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,35 @@
+expected_output = {
+    'lsps': {
+        'mlx8.1_to_ces.2': {
+            'destination': '1.1.1.1',
+            'admin': 'UP',
+            'operational': 'UP',
+            'flap_count': 1,
+            'retry_count': 0,
+            'tunnel_interface': 'tnl0'
+        },
+        'mlx8.1_to_ces.1': {
+            'destination': '2.2.2.2',
+            'admin': 'UP',
+            'operational': 'UP',
+            'flap_count': 1,
+            'retry_count': 0,
+            'tunnel_interface': 'tnl56'
+        },
+        'mlx8.1_to_mlx8.2': {
+            'destination': '3.3.3.3',
+            'admin': 'UP',
+            'operational': 'UP',
+            'flap_count': 1,
+            'retry_count': 0,
+            'tunnel_interface': 'tnl63'
```

Comment:
in some parser, short interface name converted to full interface name. I would suggest to do the same to all parsers. so that we can see full interface name all the time and easy to match/compare between parser outputs later.

Reply:
Noted, have fixed up